modules/odc-adding-health-checks.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// applications/odc-monitoring-application-health-using-developer-perspective
+
+[id="odc-adding-health-checks"]
+= Adding health checks
+
+You can use the *Topology* view to add health checks to your deployed application.
+
+.Procedure
+. In the *Topology* view, click on the application node to see the side panel. If the container does not have health checks added to ensure the smooth running of your application, a *Health Checks* notification is displayed with a link to add health checks.
+. In the displayed notification, click the *Add Health Checks* link.
+. Alternatively, you can also click the *Actions* drop-down list and select *Add Health Checks*. Note that if the container already has health checks, you will see the *Edit Health Checks* option instead of the add option.
+. In the *Add Health Checks* form, if you have deployed multiple containers, use the *Container* drop-down list to ensure that the appropriate container is selected.
+. Click the add probe links to add the required health checks. Default data for the health checks is pre-populated. You can add the probes with the default data or further customize the values and then add them. For example, to add a Readiness probe that checks if your container is ready to handle requests:
```

Comment:
I believe IBM advises against hyphening prefixes. 
I couldn't find "prepopulate" in their examples but according to quick search "prepopulate" is used more in the books/documentation in general.